
Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-7xmmd[0m
{"level":"warn","ts":1581072158.1773648,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581072158.1781368,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-wf5r5[0m
{"level":"warn","ts":1581072158.600834,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072158.6021638,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-nsh8l[0m
{"level":"warn","ts":1581072163.724344,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072163.7257202,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1415-in-ljc8v[0m
{"level":"warn","ts":1581072164.0135221,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581072167.232976,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c,1415:50f7e04316ee4f7f0bd2b59a8dd47a1762396ce6
DEBUG: ran git fetch --unshallow origin 50f7e04316ee4f7f0bd2b59a8dd47a1762396ce6: 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 9e81a0067ca9fd7a18a41b32e37e5e0e68cb076c in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 50f7e04316ee4f7f0bd2b59a8dd47a1762396ce6 in 
Merged SHA 50f7e04316ee4f7f0bd2b59a8dd47a1762396ce6 with commit message 'Restore python-builder image' into base branch master
Merged SHA 2d40890fcc77866e4a6aa2d6ea11e3bd8344018b with commit message 'Use new version of core-builder' into base branch master
Merged SHA a772cb7a49fc1fead2792172fa12fc210cac4807 with commit message 'Add manual deletion of CRD' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1415
+ BUILD_NUMBER=5
+ PIPELINE_CONTEXT=integration
++ cat
+++ date
+ COMMENT_MSG='Fri Feb  7 10:42:53 UTC 2020
The logs for [integration] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1415/5.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1415 --build=5'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1415 '--comment=Fri Feb  7 10:42:53 UTC 2020
The logs for [integration] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1415/5.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1415 --build=5'

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1415-in-mrc4f[0m
{"level":"warn","ts":1581072175.80118,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072175.8044345,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1415-in-tpzt9[0m
{"level":"warn","ts":1581072176.0901062,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072176.3239267,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-credential-initializer-f74tv[0m
{"level":"warn","ts":1581072210.478183,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1581072210.4786499,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-working-dir-initializer-9jckq[0m
{"level":"warn","ts":1581072211.712298,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072211.7135117,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-create-dir-workspace-x2wgp[0m
{"level":"warn","ts":1581072216.8220623,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072216.8233917,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-source-copy-workspace-6l5wf[0m
{"level":"warn","ts":1581072217.0896993,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1581072219.2235541,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1415-in-5rvjv-5[0m stage [32mend-to-end[0m and container [32mstep-test-end-to-end[0m
Starting Docker: docker.
[SETUP] Waiting for Docker to be ready, sleeping for 1 seconds ...
kind create cluster --config kind_config.yaml
Creating cluster "kind" ...
 • Ensuring node image (kindest/node:v1.16.3) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.16.3) 🖼
 • Preparing nodes 📦  ...
 ✓ Preparing nodes 📦
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing CNI 🔌  ...
 ✓ Installing CNI 🔌
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
 • Joining worker nodes 🚜  ...
 ✓ Joining worker nodes 🚜
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
`kind get kubeconfig-path` is deprecated!

KIND will export and merge kubeconfig like kops, minikube, etc.
This command is now unnecessary and will be removed in a future release.

For more info see: https://github.com/kubernetes-sigs/kind/issues/1060
See also the output of `kind create cluster`

Files changed in python folder:
SKIPPING PYTHON IMAGE BUILD...
Files changed in operator folder:
SKIPPING OPERATOR IMAGE BUILD...
Files changed in engine folder:
SKIPPING ENGINE IMAGE BUILD...
Files changed in executor folder:
SKIPPING EXECUTOR IMAGE BUILD...
Build fixed models
cd ../docker/fixed-model && make kind_load_images
make[1]: Entering directory '/workspace/source/testing/docker/fixed-model'
s2i build -E environment_rest_v1 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.1
---> Installing application source...
Build completed successfully
s2i build -E environment_rest_v2 . seldonio/seldon-core-s2i-python3:0.15 seldonio/fixed-model:0.2
---> Installing application source...
Build completed successfully
kind load -v 3 docker-image seldonio/fixed-model:0.1
Image: "seldonio/fixed-model:0.1" with ID "sha256:be695792ef4ba3c0c5384918eb50918427c8ee7fd51f60deb84b54cd7f2f2339" not present on node "kind-worker"
Image: "seldonio/fixed-model:0.1" with ID "sha256:be695792ef4ba3c0c5384918eb50918427c8ee7fd51f60deb84b54cd7f2f2339" not present on node "kind-control-plane"
kind load -v 3 docker-image seldonio/fixed-model:0.2
Image: "seldonio/fixed-model:0.2" with ID "sha256:8ba25b863167aebf0fb079897ce5785422a727d52a1d1710f3a24dcc236075d1" not present on node "kind-worker"
Image: "seldonio/fixed-model:0.2" with ID "sha256:8ba25b863167aebf0fb079897ce5785422a727d52a1d1710f3a24dcc236075d1" not present on node "kind-control-plane"
make[1]: Leaving directory '/workspace/source/testing/docker/fixed-model'
kubectl create namespace seldon || echo "Namespace seldon already exists"
namespace/seldon created
kubectl create namespace test1 || echo "Namespace test1 already exists"
namespace/test1 created
helm repo add stable https://kubernetes-charts.storage.googleapis.com/
"stable" has been added to your repositories
helm repo add seldonio https://storage.googleapis.com/seldon-charts
"seldonio" has been added to your repositories
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "seldonio" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈ 
helm install --wait ambassador stable/ambassador -f ambassador_values.yaml --set crds.keep=false --namespace seldon --set replicaCount=1
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
NAME: ambassador
LAST DEPLOYED: Fri Feb  7 10:49:15 2020
NAMESPACE: seldon
STATUS: deployed
REVISION: 1
NOTES:
Congratulations! You've successfully installed Ambassador.

For help, visit our Slack at https://d6e.co/slack or view the documentation online at https://www.getambassador.io.

To get the IP address of Ambassador, run the following commands:
  export NODE_PORT=$(kubectl get --namespace seldon -o jsonpath="{.spec.ports[0].nodePort}" services ambassador)
  export NODE_IP=$(kubectl get nodes --namespace seldon -o jsonpath="{.items[0].status.addresses[0].address}")
  echo http://$NODE_IP:$NODE_PORT
kubectl apply -f istio-1.4.2.yaml
clusterrole.rbac.authorization.k8s.io/istio-reader-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-reader-istio-system created
customresourcedefinition.apiextensions.k8s.io/attributemanifests.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/clusterrbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/destinationrules.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/envoyfilters.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/gateways.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/httpapispecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/meshpolicies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/policies.authentication.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecbindings.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/quotaspecs.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/rbacconfigs.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/rules.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceentries.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/servicerolebindings.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/serviceroles.rbac.istio.io created
customresourcedefinition.apiextensions.k8s.io/virtualservices.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/adapters.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/instances.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/templates.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/handlers.config.istio.io created
customresourcedefinition.apiextensions.k8s.io/sidecars.networking.istio.io created
customresourcedefinition.apiextensions.k8s.io/authorizationpolicies.security.istio.io created
namespace/istio-system created
serviceaccount/istio-reader-service-account created
clusterrole.rbac.authorization.k8s.io/istio-citadel-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-citadel-istio-system created
deployment.apps/istio-citadel created
poddisruptionbudget.policy/istio-citadel created
service/istio-citadel created
serviceaccount/istio-citadel-service-account created
deployment.apps/istio-egressgateway created
poddisruptionbudget.policy/istio-egressgateway created
gateway.networking.istio.io/istio-multicluster-egressgateway created
virtualservice.networking.istio.io/istio-multicluster-egressgateway created
envoyfilter.networking.istio.io/istio-multicluster-egressgateway created
destinationrule.networking.istio.io/istio-multicluster-destinationrule created
service/istio-egressgateway created
role.rbac.authorization.k8s.io/istio-egressgateway-sds created
rolebinding.rbac.authorization.k8s.io/istio-egressgateway-sds created
serviceaccount/istio-egressgateway-service-account created
clusterrole.rbac.authorization.k8s.io/istio-galley-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-galley-*****-role-binding-istio-system created
configmap/istio-mesh-galley created
configmap/istio-galley-configuration created
deployment.apps/istio-galley created
poddisruptionbudget.policy/istio-galley created
service/istio-galley created
serviceaccount/istio-galley-service-account created
configmap/istio-grafana-configuration-dashboards-citadel-dashboard created
configmap/istio-grafana-configuration-dashboards-galley-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-mesh-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-performance-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-service-dashboard created
configmap/istio-grafana-configuration-dashboards-istio-workload-dashboard created
configmap/istio-grafana-configuration-dashboards-mixer-dashboard created
configmap/istio-grafana-configuration-dashboards-pilot-dashboard created
configmap/istio-grafana created
deployment.apps/grafana created
policy.authentication.istio.io/grafana-ports-mtls-disabled created
service/grafana created
deployment.apps/istio-ingressgateway created
gateway.networking.istio.io/ingressgateway created
poddisruptionbudget.policy/ingressgateway created
service/istio-ingressgateway created
serviceaccount/istio-ingressgateway-service-account created
sidecar.networking.istio.io/default created
clusterrole.rbac.authorization.k8s.io/istio-sidecar-injector-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-sidecar-injector-*****-role-binding-istio-system created
configmap/injector-mesh created
deployment.apps/istio-sidecar-injector created
mutatingwebhookconfiguration.admissionregistration.k8s.io/istio-sidecar-injector created
poddisruptionbudget.policy/istio-sidecar-injector created
service/istio-sidecar-injector created
serviceaccount/istio-sidecar-injector-service-account created
configmap/istio-sidecar-injector created
clusterrole.rbac.authorization.k8s.io/kiali created
clusterrole.rbac.authorization.k8s.io/kiali-viewer created
clusterrolebinding.rbac.authorization.k8s.io/kiali created
configmap/kiali created
secret/kiali created
deployment.apps/kiali created
service/kiali created
serviceaccount/kiali-service-account created
clusterrole.rbac.authorization.k8s.io/istio-pilot-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-pilot-istio-system created
configmap/pilot-envoy-config created
configmap/istio created
deployment.apps/istio-pilot created
meshpolicy.authentication.istio.io/default created
poddisruptionbudget.policy/istio-pilot created
service/istio-pilot created
serviceaccount/istio-pilot-service-account created
clusterrole.rbac.authorization.k8s.io/istio-policy created
clusterrolebinding.rbac.authorization.k8s.io/istio-policy-*****-role-binding-istio-system created
destinationrule.networking.istio.io/istio-policy created
deployment.apps/istio-policy created
poddisruptionbudget.policy/istio-policy created
service/istio-policy created
serviceaccount/istio-policy-service-account created
clusterrole.rbac.authorization.k8s.io/prometheus-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/prometheus-istio-system created
configmap/prometheus created
deployment.apps/prometheus created
service/prometheus created
serviceaccount/prometheus created
horizontalpodautoscaler.autoscaling/istio-telemetry created
clusterrole.rbac.authorization.k8s.io/istio-mixer-istio-system created
clusterrolebinding.rbac.authorization.k8s.io/istio-mixer-*****-role-binding-istio-system created
attributemanifest.config.istio.io/istioproxy created
attributemanifest.config.istio.io/kubernetes created
handler.config.istio.io/stdio created
instance.config.istio.io/accesslog created
instance.config.istio.io/tcpaccesslog created
rule.config.istio.io/stdio created
rule.config.istio.io/stdiotcp created
instance.config.istio.io/requestcount created
instance.config.istio.io/requestduration created
instance.config.istio.io/requestsize created
instance.config.istio.io/responsesize created
instance.config.istio.io/tcpbytesent created
instance.config.istio.io/tcpbytereceived created
instance.config.istio.io/tcpconnectionsopened created
instance.config.istio.io/tcpconnectionsclosed created
handler.config.istio.io/prometheus created
rule.config.istio.io/promhttp created
rule.config.istio.io/promtcp created
rule.config.istio.io/promtcpconnectionopen created
rule.config.istio.io/promtcpconnectionclosed created
handler.config.istio.io/kubernetesenv created
rule.config.istio.io/kubeattrgenrulerule created
rule.config.istio.io/tcpkubeattrgenrulerule created
instance.config.istio.io/attributes created
destinationrule.networking.istio.io/istio-telemetry created
configmap/telemetry-envoy-config created
deployment.apps/istio-telemetry created
poddisruptionbudget.policy/istio-telemetry created
service/istio-telemetry created
serviceaccount/istio-mixer-service-account created
deployment.apps/istio-tracing created
service/jaeger-query created
service/jaeger-collector created
service/jaeger-agent created
service/zipkin created
service/tracing created
kubectl rollout status deployment.apps/istio-ingressgateway -n istio-system
Waiting for deployment "istio-ingressgateway" rollout to finish: 0 of 1 updated replicas are available...
deployment "istio-ingressgateway" successfully rolled out
kubectl rollout status deployment.apps/istio-pilot -n istio-system
deployment "istio-pilot" successfully rolled out
kubectl rollout status deployment.apps/istio-citadel -n istio-system
deployment "istio-citadel" successfully rolled out
cd ../../operator && make install-cert-manager
make[1]: Entering directory '/workspace/source/operator'
kubectl create namespace cert-manager || echo "Namespace cert-manager-exists"
namespace/cert-manager created
kubectl label namespace cert-manager cert-manager.io/disable-validation=true || echo "namespace cert-manager-already labelled"
namespace/cert-manager labeled
kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.12.0/cert-manager.yaml
customresourcedefinition.apiextensions.k8s.io/certificaterequests.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/certificates.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/challenges.acme.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/clusterissuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/issuers.cert-manager.io created
customresourcedefinition.apiextensions.k8s.io/orders.acme.cert-manager.io created
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
namespace/cert-manager configured
serviceaccount/cert-manager-cainjector created
serviceaccount/cert-manager created
serviceaccount/cert-manager-webhook created
clusterrole.rbac.authorization.k8s.io/cert-manager-cainjector created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-cainjector created
role.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager-cainjector:leaderelection created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-webhook:auth-delegator created
rolebinding.rbac.authorization.k8s.io/cert-manager-webhook:webhook-authentication-reader created
clusterrole.rbac.authorization.k8s.io/cert-manager-webhook:webhook-requester created
role.rbac.authorization.k8s.io/cert-manager:leaderelection created
rolebinding.rbac.authorization.k8s.io/cert-manager:leaderelection created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrole.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-issuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-clusterissuers created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-certificates created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-orders created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-challenges created
clusterrolebinding.rbac.authorization.k8s.io/cert-manager-controller-ingress-shim created
clusterrole.rbac.authorization.k8s.io/cert-manager-view created
clusterrole.rbac.authorization.k8s.io/cert-manager-edit created
service/cert-manager created
service/cert-manager-webhook created
deployment.apps/cert-manager-cainjector created
deployment.apps/cert-manager created
deployment.apps/cert-manager-webhook created
mutatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
validatingwebhookconfiguration.admissionregistration.k8s.io/cert-manager-webhook created
kubectl rollout status deployment.apps/cert-manager -n cert-manager
Waiting for deployment "cert-manager" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-cainjector -n cert-manager
deployment "cert-manager-cainjector" successfully rolled out
kubectl rollout status deployment.apps/cert-manager-webhook -n cert-manager
Waiting for deployment "cert-manager-webhook" rollout to finish: 0 of 1 updated replicas are available...
deployment "cert-manager-webhook" successfully rolled out
make[1]: Leaving directory '/workspace/source/operator'
sleep 5 #https://github.com/jetstack/cert-manager/issues/2273
kubectl create namespace seldon-system || echo "namespace seldon-system exists"
namespace/seldon-system created
helm install --wait seldon ../../helm-charts/seldon-core-operator --namespace seldon-system --set istio.enabled=true --set istio.gateway=seldon-gateway --set certManager.enabled=false --set executor.enabled=true
NAME: seldon
LAST DEPLOYED: Fri Feb  7 10:52:28 2020
NAMESPACE: seldon-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
kubectl config set-context $(kubectl config current-context) --namespace=seldon
Context "kind-kind" modified.
make: Entering directory '/workspace/source/python'
pip install -e . -r requirements-dev.txt
Obtaining file:///workspace/source/python
Collecting Flask<2.0.0
  Downloading Flask-1.1.1-py2.py3-none-any.whl (94 kB)
Collecting Flask-cors<4.0.0
  Downloading Flask_Cors-3.0.8-py2.py3-none-any.whl (14 kB)
Collecting redis<4.0.0
  Downloading redis-3.4.1-py2.py3-none-any.whl (71 kB)
Collecting requests<3.0.0
  Downloading requests-2.22.0-py2.py3-none-any.whl (57 kB)
Collecting numpy<2.0.0
  Downloading numpy-1.18.1-cp36-cp36m-manylinux1_x86_64.whl (20.1 MB)
Collecting flatbuffers<2.0.0
  Downloading flatbuffers-1.11-py2.py3-none-any.whl (15 kB)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 7)) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 8)) (1.26.0)
Collecting Flask-OpenTracing<1.2.0,>=1.1.0
  Downloading Flask-OpenTracing-1.1.0.tar.gz (8.2 kB)
Collecting opentracing<2.3.0,>=2.2.0
  Downloading opentracing-2.2.0.tar.gz (47 kB)
Collecting jaeger-client<4.2.0,>=4.1.0
  Downloading jaeger-client-4.1.0.tar.gz (80 kB)
Collecting grpcio-opentracing<1.2.0,>=1.1.4
  Downloading grpcio_opentracing-1.1.4-py3-none-any.whl (14 kB)
Collecting pyaml<20.0.0
  Downloading pyaml-19.12.0-py2.py3-none-any.whl (17 kB)
Collecting gunicorn<20.1.0,>=19.9.0
  Downloading gunicorn-20.0.4-py2.py3-none-any.whl (77 kB)
Collecting minio<6.0.0,>=4.0.9
  Downloading minio-5.0.7-py2.py3-none-any.whl (71 kB)
Collecting azure-storage-blob<3.0.0,>=2.0.1
  Downloading azure_storage_blob-2.1.0-py2.py3-none-any.whl (88 kB)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 17)) (45.1.0)
Collecting pandas==0.25.3
  Downloading pandas-0.25.3-cp36-cp36m-manylinux1_x86_64.whl (10.4 MB)
Collecting google-cloud-storage>=1.16.0
  Downloading google_cloud_storage-1.25.0-py2.py3-none-any.whl (73 kB)
Collecting mypy<0.762
  Downloading mypy-0.761-cp36-cp36m-manylinux1_x86_64.whl (24.1 MB)
Collecting Pillow==7.0.0
  Downloading Pillow-7.0.0-cp36-cp36m-manylinux1_x86_64.whl (2.1 MB)
Collecting tox<4.0.0
  Downloading tox-3.14.3-py2.py3-none-any.whl (80 kB)
Collecting pytest==5.3.1
  Downloading pytest-5.3.1-py3-none-any.whl (233 kB)
Collecting pytest-cov==2.8.1
  Downloading pytest_cov-2.8.1-py2.py3-none-any.whl (18 kB)
Collecting coverage==4.5.4
  Downloading coverage-4.5.4-cp36-cp36m-manylinux1_x86_64.whl (205 kB)
Collecting itsdangerous>=0.24
  Downloading itsdangerous-1.1.0-py2.py3-none-any.whl (16 kB)
Collecting Jinja2>=2.10.1
  Downloading Jinja2-2.11.1-py2.py3-none-any.whl (126 kB)
Collecting click>=5.1
  Downloading Click-7.0-py2.py3-none-any.whl (81 kB)
Collecting Werkzeug>=0.15
  Downloading Werkzeug-1.0.0-py2.py3-none-any.whl (298 kB)
Requirement already satisfied: Six in /usr/local/lib/python3.6/site-packages (from Flask-cors<4.0.0->-r requirements.txt (line 2)) (1.14.0)
Collecting chardet<3.1.0,>=3.0.2
  Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2019.11.28-py2.py3-none-any.whl (156 kB)
Collecting idna<2.9,>=2.5
  Downloading idna-2.8-py2.py3-none-any.whl (58 kB)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1
  Downloading urllib3-1.25.8-py2.py3-none-any.whl (125 kB)
Collecting threadloop<2,>=1
  Downloading threadloop-1.0.2.tar.gz (4.9 kB)
Collecting thrift
  Downloading thrift-0.13.0.tar.gz (59 kB)
Collecting tornado<6,>=4.3
  Downloading tornado-5.1.1.tar.gz (516 kB)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->-r requirements.txt (line 13)) (5.3)
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting pytz
  Downloading pytz-2019.3-py2.py3-none-any.whl (509 kB)
Collecting configparser
  Downloading configparser-4.0.2-py2.py3-none-any.whl (22 kB)
Collecting azure-common>=1.1.5
  Downloading azure_common-1.1.24-py2.py3-none-any.whl (12 kB)
Collecting azure-storage-common~=2.1
  Downloading azure_storage_common-2.1.0-py2.py3-none-any.whl (47 kB)
Collecting google-resumable-media<0.6dev,>=0.5.0
  Downloading google_resumable_media-0.5.0-py2.py3-none-any.whl (38 kB)
Collecting google-auth<2.0dev,>=1.9.0
  Downloading google_auth-1.11.0-py2.py3-none-any.whl (76 kB)
Collecting google-cloud-core<2.0dev,>=1.2.0
  Downloading google_cloud_core-1.3.0-py2.py3-none-any.whl (26 kB)
Collecting typed-ast<1.5.0,>=1.4.0
  Downloading typed_ast-1.4.1-cp36-cp36m-manylinux1_x86_64.whl (737 kB)
Collecting mypy-extensions<0.5.0,>=0.4.3
  Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Collecting typing-extensions>=3.7.4
  Downloading typing_extensions-3.7.4.1-py3-none-any.whl (20 kB)
Collecting filelock<4,>=3.0.0
  Downloading filelock-3.0.12-py3-none-any.whl (7.6 kB)
Collecting importlib-metadata<2,>=0.12; python_version < "3.8"
  Downloading importlib_metadata-1.5.0-py2.py3-none-any.whl (30 kB)
Collecting virtualenv>=16.0.0
  Downloading virtualenv-16.7.9-py2.py3-none-any.whl (3.4 MB)
Collecting toml>=0.9.4
  Downloading toml-0.10.0-py2.py3-none-any.whl (25 kB)
Collecting py<2,>=1.4.17
  Downloading py-1.8.1-py2.py3-none-any.whl (83 kB)
Collecting packaging>=14
  Downloading packaging-20.1-py2.py3-none-any.whl (36 kB)
Collecting pluggy<1,>=0.12.0
  Downloading pluggy-0.13.1-py2.py3-none-any.whl (18 kB)
Collecting more-itertools>=4.0.0
  Downloading more_itertools-8.2.0-py3-none-any.whl (43 kB)
Collecting wcwidth
  Downloading wcwidth-0.1.8-py2.py3-none-any.whl (17 kB)
Collecting attrs>=17.4.0
  Downloading attrs-19.3.0-py2.py3-none-any.whl (39 kB)
Collecting MarkupSafe>=0.23
  Downloading MarkupSafe-1.1.1-cp36-cp36m-manylinux1_x86_64.whl (27 kB)
Collecting cryptography
  Downloading cryptography-2.8-cp34-abi3-manylinux2010_x86_64.whl (2.3 MB)
Collecting cachetools<5.0,>=2.0.0
  Downloading cachetools-4.0.0-py3-none-any.whl (10 kB)
Collecting rsa<4.1,>=3.1.4
  Downloading rsa-4.0-py2.py3-none-any.whl (38 kB)
Collecting pyasn1-modules>=0.2.1
  Downloading pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
Collecting google-api-core<2.0.0dev,>=1.16.0
  Downloading google_api_core-1.16.0-py2.py3-none-any.whl (70 kB)
Collecting zipp>=0.5
  Downloading zipp-2.1.0-py3-none-any.whl (4.6 kB)
Collecting pyparsing>=2.0.2
  Downloading pyparsing-2.4.6-py2.py3-none-any.whl (67 kB)
Collecting cffi!=1.11.3,>=1.8
  Downloading cffi-1.13.2-cp36-cp36m-manylinux1_x86_64.whl (397 kB)
Collecting pyasn1>=0.1.3
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Collecting googleapis-common-protos<2.0dev,>=1.6.0
  Downloading googleapis-common-protos-1.51.0.tar.gz (35 kB)
Collecting pycparser
  Downloading pycparser-2.19.tar.gz (158 kB)
Building wheels for collected packages: Flask-OpenTracing, opentracing, jaeger-client, threadloop, thrift, tornado, googleapis-common-protos, pycparser
  Building wheel for Flask-OpenTracing (setup.py): started
  Building wheel for Flask-OpenTracing (setup.py): finished with status 'done'
  Created wheel for Flask-OpenTracing: filename=Flask_OpenTracing-1.1.0-py3-none-any.whl size=9070 sha256=d20dcfff1adbdbdcc4ffe5d637a93395b826497150369c1af7979671e9dad61d
  Stored in directory: /builder/home/.cache/pip/wheels/ad/4b/2d/24ff0da0a0b53c7c77ce59b843bcceaf644c88703241e59615
  Building wheel for opentracing (setup.py): started
  Building wheel for opentracing (setup.py): finished with status 'done'
  Created wheel for opentracing: filename=opentracing-2.2.0-py3-none-any.whl size=49319 sha256=9e18f16d15eb0ac6110583ae67c9cacdc66fe9fbfb6d58e8b8e1253a3c063f37
  Stored in directory: /builder/home/.cache/pip/wheels/39/40/44/8bace79f4514e99786236c31f1df8d1b814ff02c1e08b1d697
  Building wheel for jaeger-client (setup.py): started
  Building wheel for jaeger-client (setup.py): finished with status 'done'
  Created wheel for jaeger-client: filename=jaeger_client-4.1.0-py3-none-any.whl size=64309 sha256=c8220face8eb273dc60b4d046a2f303e94efa74659100a025d9729d43e0af8e6
  Stored in directory: /builder/home/.cache/pip/wheels/e9/9b/8c/503d0cc13b39a551c054515683ba1d15b40324c863dc442e66
  Building wheel for threadloop (setup.py): started
  Building wheel for threadloop (setup.py): finished with status 'done'
  Created wheel for threadloop: filename=threadloop-1.0.2-py3-none-any.whl size=3423 sha256=6ec1b2cb740b36f10b2f4e143ac91b9df2e759d61fddd18296ce2dee9930df77
  Stored in directory: /builder/home/.cache/pip/wheels/02/54/65/9f87de48fe8fcaaee30f279973d946ad55f9df56b93b3e78da
  Building wheel for thrift (setup.py): started
  Building wheel for thrift (setup.py): finished with status 'done'
  Created wheel for thrift: filename=thrift-0.13.0-cp36-cp36m-linux_x86_64.whl size=483899 sha256=f916971d795c9276e5fea9709d4f3d672a7e52bb57a3aa079f01865554e8f635
  Stored in directory: /builder/home/.cache/pip/wheels/e0/38/fc/472fe18756b177b42096961f8bd3ff2dc5c5620ac399fce52d
  Building wheel for tornado (setup.py): started
  Building wheel for tornado (setup.py): finished with status 'done'
  Created wheel for tornado: filename=tornado-5.1.1-cp36-cp36m-linux_x86_64.whl size=462650 sha256=0ade213a4bf44c117ce12fd46fc40b57305ee4b627b97a8846237598a0bb298e
  Stored in directory: /builder/home/.cache/pip/wheels/64/74/71/e7a0d0eb4fc42cb20a17aec36f8f0b5b8c5e1ec19c701fd34a
  Building wheel for googleapis-common-protos (setup.py): started
  Building wheel for googleapis-common-protos (setup.py): finished with status 'done'
  Created wheel for googleapis-common-protos: filename=googleapis_common_protos-1.51.0-py3-none-any.whl size=77592 sha256=390291d336fdbcb727006243e70c4b836c9f3fea82865a4a66a00799b56e222b
  Stored in directory: /builder/home/.cache/pip/wheels/35/8d/af/a922cb18800b31fadac3523cadf6c1efdf233b788fe7a4da70
  Building wheel for pycparser (setup.py): started
  Building wheel for pycparser (setup.py): finished with status 'done'
  Created wheel for pycparser: filename=pycparser-2.19-py2.py3-none-any.whl size=111031 sha256=809029042454174d7957ce525973fbe0ee1c57deee3bd56c2df6c4f878b5b4e1
  Stored in directory: /builder/home/.cache/pip/wheels/c6/6b/83/2608afaa57ecfb0a66ac89191a8d9bad71c62ca55ee499c2d0
Successfully built Flask-OpenTracing opentracing jaeger-client threadloop thrift tornado googleapis-common-protos pycparser
Installing collected packages: itsdangerous, MarkupSafe, Jinja2, click, Werkzeug, Flask, Flask-cors, redis, chardet, certifi, idna, urllib3, requests, numpy, flatbuffers, opentracing, Flask-OpenTracing, tornado, threadloop, thrift, jaeger-client, grpcio-opentracing, pyaml, gunicorn, python-dateutil, pytz, configparser, minio, azure-common, pycparser, cffi, cryptography, azure-storage-common, azure-storage-blob, pandas, google-resumable-media, cachetools, pyasn1, rsa, pyasn1-modules, google-auth, googleapis-common-protos, google-api-core, google-cloud-core, google-cloud-storage, typed-ast, mypy-extensions, typing-extensions, mypy, Pillow, filelock, zipp, importlib-metadata, virtualenv, toml, py, pyparsing, packaging, pluggy, tox, more-itertools, wcwidth, attrs, pytest, coverage, pytest-cov, seldon-core
  Running setup.py develop for seldon-core
Successfully installed Flask-1.1.1 Flask-OpenTracing-1.1.0 Flask-cors-3.0.8 Jinja2-2.11.1 MarkupSafe-1.1.1 Pillow-7.0.0 Werkzeug-1.0.0 attrs-19.3.0 azure-common-1.1.24 azure-storage-blob-2.1.0 azure-storage-common-2.1.0 cachetools-4.0.0 certifi-2019.11.28 cffi-1.13.2 chardet-3.0.4 click-7.0 configparser-4.0.2 coverage-4.5.4 cryptography-2.8 filelock-3.0.12 flatbuffers-1.11 google-api-core-1.16.0 google-auth-1.11.0 google-cloud-core-1.3.0 google-cloud-storage-1.25.0 google-resumable-media-0.5.0 googleapis-common-protos-1.51.0 grpcio-opentracing-1.1.4 gunicorn-20.0.4 idna-2.8 importlib-metadata-1.5.0 itsdangerous-1.1.0 jaeger-client-4.1.0 minio-5.0.7 more-itertools-8.2.0 mypy-0.761 mypy-extensions-0.4.3 numpy-1.18.1 opentracing-2.2.0 packaging-20.1 pandas-0.25.3 pluggy-0.13.1 py-1.8.1 pyaml-19.12.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pycparser-2.19 pyparsing-2.4.6 pytest-5.3.1 pytest-cov-2.8.1 python-dateutil-2.8.1 pytz-2019.3 redis-3.4.1 requests-2.22.0 rsa-4.0 seldon-core threadloop-1.0.2 thrift-0.13.0 toml-0.10.0 tornado-5.1.1 tox-3.14.3 typed-ast-1.4.1 typing-extensions-3.7.4.1 urllib3-1.25.8 virtualenv-16.7.9 wcwidth-0.1.8 zipp-2.1.0
make: Leaving directory '/workspace/source/python'
cp ../../proto/prediction.proto ./proto
cd ../../proto/tensorflow && make create_protos
make[1]: Entering directory '/workspace/source/proto/tensorflow'
./create-protos.sh
Downloading proto files for master
make[1]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../../proto/tensorflow/tensorflow .
'../../proto/tensorflow/tensorflow' -> './tensorflow'
'../../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
cd ../../notebooks && make build_protos
make[1]: Entering directory '/workspace/source/notebooks'
cd ../proto/tensorflow && make create_protos
make[2]: Entering directory '/workspace/source/proto/tensorflow'
make[2]: Nothing to be done for 'create_protos'.
make[2]: Leaving directory '/workspace/source/proto/tensorflow'
cp -vr ../proto/tensorflow/tensorflow .
'../proto/tensorflow/tensorflow' -> './tensorflow'
'../proto/tensorflow/tensorflow/core' -> './tensorflow/core'
'../proto/tensorflow/tensorflow/core/framework' -> './tensorflow/core/framework'
'../proto/tensorflow/tensorflow/core/framework/types.proto' -> './tensorflow/core/framework/types.proto'
'../proto/tensorflow/tensorflow/core/framework/resource_handle.proto' -> './tensorflow/core/framework/resource_handle.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor_shape.proto' -> './tensorflow/core/framework/tensor_shape.proto'
'../proto/tensorflow/tensorflow/core/framework/tensor.proto' -> './tensorflow/core/framework/tensor.proto'
cp ../proto/prediction.proto ./proto
python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. ./proto/prediction.proto
make[1]: Leaving directory '/workspace/source/notebooks'
cd ../../proto/k8s && make create_protos
make[1]: Entering directory '/workspace/source/proto/k8s'
./create-k8s-protos.sh
Downloading proto files for master
Munging proto file packages
sed -i.bak 's|import "k8s.io/apiextensions-apiserver/|//import "k8s.io/apiextensions-apiserver/|' k8s.io/api/core/v1/generated.proto
rm k8s.io/api/core/v1/generated.proto.bak
make[1]: Leaving directory '/workspace/source/proto/k8s'
pip install -r dev_requirements.txt
Obtaining file:///workspace/source/python (from -r dev_requirements.txt (line 8))
Requirement already satisfied: pytest==5.3.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 1)) (5.3.1)
Collecting pytest-xdist==1.30.0
  Downloading pytest_xdist-1.30.0-py2.py3-none-any.whl (35 kB)
Requirement already satisfied: pytest-cov==2.8.1 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 3)) (2.8.1)
Requirement already satisfied: coverage==4.5.4 in /usr/local/lib/python3.6/site-packages (from -r dev_requirements.txt (line 5)) (4.5.4)
Collecting retrying
  Downloading retrying-1.3.3.tar.gz (10 kB)
Requirement already satisfied: Flask<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.1)
Requirement already satisfied: Flask-cors<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (3.0.8)
Requirement already satisfied: redis<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (3.4.1)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.22.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.18.1)
Requirement already satisfied: flatbuffers<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.11)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (3.11.3)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.26.0)
Requirement already satisfied: Flask-OpenTracing<1.2.0,>=1.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.0)
Requirement already satisfied: opentracing<2.3.0,>=2.2.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.2.0)
Requirement already satisfied: jaeger-client<4.2.0,>=4.1.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (4.1.0)
Requirement already satisfied: grpcio-opentracing<1.2.0,>=1.1.4 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.4)
Requirement already satisfied: pyaml<20.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (19.12.0)
Requirement already satisfied: gunicorn<20.1.0,>=19.9.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (20.0.4)
Requirement already satisfied: minio<6.0.0,>=4.0.9 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (5.0.7)
Requirement already satisfied: azure-storage-blob<3.0.0,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.1.0)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (45.1.0)
Requirement already satisfied: attrs>=17.4.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (19.3.0)
Requirement already satisfied: more-itertools>=4.0.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (8.2.0)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (20.1)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.1.8)
Requirement already satisfied: importlib-metadata>=0.12; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.5.0)
Requirement already satisfied: py>=1.5.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (1.8.1)
Requirement already satisfied: pluggy<1.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest==5.3.1->-r dev_requirements.txt (line 1)) (0.13.1)
Requirement already satisfied: six in /usr/local/lib/python3.6/site-packages (from pytest-xdist==1.30.0->-r dev_requirements.txt (line 2)) (1.14.0)
Collecting pytest-forked
  Downloading pytest_forked-1.1.3-py2.py3-none-any.whl (4.5 kB)
Collecting execnet>=1.1
  Downloading execnet-1.7.1-py2.py3-none-any.whl (39 kB)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.0.0)
Requirement already satisfied: click>=5.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (7.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.11.1)
Requirement already satisfied: itsdangerous>=0.24 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.0)
Requirement already satisfied: idna<2.9,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.8)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2019.11.28)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (3.0.4)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.25.8)
Requirement already satisfied: thrift in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (0.13.0)
Requirement already satisfied: tornado<6,>=4.3 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (5.1.1)
Requirement already satisfied: threadloop<2,>=1 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.0.2)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/site-packages (from pyaml<20.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (5.3)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.8.1)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2019.3)
Requirement already satisfied: configparser in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (4.0.2)
Requirement already satisfied: azure-common>=1.1.5 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.24)
Requirement already satisfied: azure-storage-common~=2.1 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.1.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->pytest==5.3.1->-r dev_requirements.txt (line 1)) (2.4.6)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata>=0.12; python_version < "3.8"->pytest==5.3.1->-r dev_requirements.txt (line 1)) (2.1.0)
Collecting apipkg>=1.4
  Downloading apipkg-1.5-py2.py3-none-any.whl (4.9 kB)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.1.1)
Requirement already satisfied: cryptography in /usr/local/lib/python3.6/site-packages (from azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.8)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib/python3.6/site-packages (from cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (1.13.2)
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.8->cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->seldon-core==1.0.1->-r dev_requirements.txt (line 8)) (2.19)
Building wheels for collected packages: retrying
  Building wheel for retrying (setup.py): started
  Building wheel for retrying (setup.py): finished with status 'done'
  Created wheel for retrying: filename=retrying-1.3.3-py3-none-any.whl size=11430 sha256=6aa897d1ff0d37be41046c6bd0e335e87610cbaada646fd3e4645ea8e65990d5
  Stored in directory: /builder/home/.cache/pip/wheels/ac/cb/8a/b27bf6323e2f4c462dcbf77d70b7c5e7868a7fbe12871770cf
Successfully built retrying
Installing collected packages: pytest-forked, apipkg, execnet, pytest-xdist, retrying, seldon-core
  Attempting uninstall: seldon-core
    Found existing installation: seldon-core 1.0.1
    Uninstalling seldon-core-1.0.1:
      Successfully uninstalled seldon-core-1.0.1
  Running setup.py develop for seldon-core
Successfully installed apipkg-1.5 execnet-1.7.1 pytest-forked-1.1.3 pytest-xdist-1.30.0 retrying-1.3.3 seldon-core
# Run the core tests in parallel
pytest \
	--verbose \
	-s \
	-W ignore \
	-n "4" \
	-m "not sequential" 2>&1
============================= test session starts ==============================
platform linux -- Python 3.6.0, pytest-5.3.1, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /workspace/source/testing/scripts, inifile: setup.cfg
plugins: xdist-1.30.0, forked-1.1.3, cov-2.8.1
gw0 I / gw1 I / gw2 I / gw3 I
[gw0] linux Python 3.6.0 cwd: /workspace/source/testing/scripts
[gw1] linux Python 3.6.0 cwd: /workspace/source/testing/scripts
[gw2] linux Python 3.6.0 cwd: /workspace/source/testing/scripts
[gw3] linux Python 3.6.0 cwd: /workspace/source/testing/scripts
[gw0] Python 3.6.0 (default, Feb  4 2020, 10:38:29)  -- [GCC 8.3.0]
[gw1] Python 3.6.0 (default, Feb  4 2020, 10:38:29)  -- [GCC 8.3.0]
[gw2] Python 3.6.0 (default, Feb  4 2020, 10:38:29)  -- [GCC 8.3.0]
[gw3] Python 3.6.0 (default, Feb  4 2020, 10:38:29)  -- [GCC 8.3.0]
gw0 [36] / gw1 [36] / gw2 [36] / gw3 [36]

scheduling tests via LoadScheduling

test_api_version.py::test_api_version[machinelearning.seldon.io/v1] 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] 
test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] 
test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name Error from server (SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "dupname": Traffic must sum to 100 for multiple predictors]): error when creating "../resources/bad_duplicate_predictor_name.json": admission webhook "vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "dupname" is invalid: [spec.predictors[1]: Invalid value: "mymodel": Duplicate predictor name, spec: Invalid value: "dupname": Traffic must sum to 100 for multiple predictors]

[gw3] PASSED test_bad_graphs.py::TestBadGraphs::test_duplicate_predictor_name 
test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model Error: uninstall: Release not loaded: mymodel: release: not found

[gw1] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha3] Error: uninstall: Release not loaded: mymodel: release: not found

[gw0] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1alpha2] Error: uninstall: Release not loaded: mymab: release: not found

[gw3] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_mab_model Error: uninstall: Release not loaded: mymodel: release: not found

[gw2] PASSED test_api_version.py::test_api_version[machinelearning.seldon.io/v1] 
test_helm_charts_clusterwide.py::TestClusterWide::test_single_model 
test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model 
test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch Error from server (SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit): error when creating "../resources/bad_name_mismatch.json": admission webhook "vseldondeployment.kb.io" denied the request: SeldonDeployment.machinelearing.seldon.io "namemismatch" is invalid: spec.predictors[0].graph: Invalid value: "complex-model_bad_name": Can't find container for Predictive Unit

[gw0] PASSED test_bad_graphs.py::TestBadGraphs::test_model_name_mismatch 
test_local_operators.py::TestLocalOperators::test_namespace_operator 
test_local_operators.py::TestLocalOperators::test_labelled_operator Error from server (InternalError): error when creating "../resources/model_controller_id.yaml": Internal error occurred: failed calling webhook "mseldondeployment.kb.io": Post https://seldon-webhook-service.test-labelled-operator.svc:443/mutate-machinelearning-seldon-io-v1alpha2-seldondeployment?timeout=30s: dial tcp 10.107.100.79:443: connect: connection refused
Error: uninstall: Release not loaded: mymodel: release: not found

[gw1] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_single_model Error: uninstall: Release not loaded: myabtest: release: not found

[gw2] PASSED test_helm_charts_clusterwide.py::TestClusterWide::test_abtest_model 
[gw0] PASSED test_local_operators.py::TestLocalOperators::test_namespace_operator 
[gw3] PASSED test_local_operators.py::TestLocalOperators::test_labelled_operator 
test_prepackaged_servers.py::TestPrepack::test_tfserving 
test_rolling_updates.py::TestRollingHttp::test_rolling_update1[ambas] 
test_prepackaged_servers.py::TestPrepack::test_mlflow 
test_prepackaged_servers.py::TestPrepack::test_sklearn Error from server (NotFound): error when deleting "../resources/graph2.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update1[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update1[istio] 
[gw2] PASSED test_prepackaged_servers.py::TestPrepack::test_mlflow 
[gw3] PASSED test_prepackaged_servers.py::TestPrepack::test_sklearn 
test_rolling_updates.py::TestRollingHttp::test_rolling_update2[ambas] Error from server (NotFound): error when deleting "../resources/graph2.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update1[istio] 
[gw1] PASSED test_prepackaged_servers.py::TestPrepack::test_tfserving 
test_rolling_updates.py::TestRollingHttp::test_rolling_update2[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update3[ambas] 
test_prepackaged_servers.py::TestPrepack::test_xgboost 
[gw2] FAILED test_rolling_updates.py::TestRollingHttp::test_rolling_update2[ambas] Error from server (NotFound): error when deleting "../resources/graph3.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw3] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update2[istio] 
[gw1] PASSED test_prepackaged_servers.py::TestPrepack::test_xgboost 
test_rolling_updates.py::TestRollingHttp::test_rolling_update3[istio] Error from server (NotFound): error when deleting "../resources/graph4.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update3[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update5[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update4[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update4[istio] Error from server (NotFound): error when deleting "../resources/graph4.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw2] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update3[istio] Error from server (NotFound): error when deleting "../resources/graph6.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw1] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update5[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update6[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update5[istio] Error from server (NotFound): error when deleting "../resources/graph5.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw3] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update4[ambas] Error from server (NotFound): error when deleting "../resources/graph5.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update4[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update6[istio] Error from server (NotFound): error when deleting "../resources/graph2svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw1] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update6[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update7[ambas] Error from server (NotFound): error when deleting "../resources/graph6.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw2] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update5[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update7[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update8[ambas] Error from server (NotFound): error when deleting "../resources/graph2svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw3] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update6[istio] 
[gw0] FAILED test_rolling_updates.py::TestRollingHttp::test_rolling_update7[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update8[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update9[ambas] Error from server (NotFound): error when deleting "../resources/graph3svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw1] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update7[istio] Error from server (NotFound): error when deleting "../resources/graph4svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw2] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update8[ambas] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update9[istio] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update10[ambas] Error from server (NotFound): error when deleting "../resources/graph5svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update9[ambas] Error from server (NotFound): error when deleting "../resources/graph4svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw3] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update8[istio] 
test_rolling_updates.py::test_rolling_update_deployment[graph1.json-graph8.json] 
test_rolling_updates.py::TestRollingHttp::test_rolling_update10[istio] Error from server (NotFound): error when deleting "../resources/graph5svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw1] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update9[istio] Error from server (NotFound): error when deleting "../resources/graph6svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw2] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update10[ambas] 
test_rolling_updates.py::test_rolling_update_deployment[graph7.json-graph8.json] Error from server (NotFound): error when deleting "../resources/graph6svc.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw3] PASSED test_rolling_updates.py::TestRollingHttp::test_rolling_update10[istio] Error from server (NotFound): error when deleting "../resources/graph8.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw0] PASSED test_rolling_updates.py::test_rolling_update_deployment[graph1.json-graph8.json] Error from server (NotFound): error when deleting "../resources/graph8.json": seldondeployments.machinelearning.seldon.io "mymodel" not found

[gw1] PASSED test_rolling_updates.py::test_rolling_update_deployment[graph7.json-graph8.json] 

=================================== FAILURES ===================================
_________________ TestRollingHttp.test_rolling_update2[ambas] __________________
[gw2] linux -- Python 3.6.0 /usr/local/bin/python

self = <test_rolling_updates.TestRollingHttp object at 0x7fc7ae3e0e10>
namespace = 'test-rolling-update2-ambas', api_gateway = 'localhost:8003'

    @with_api_gateways
    # test changing the image version and the name of its container
    def test_rolling_update2(self, namespace, api_gateway):
        if api_gateway == API_ISTIO_GATEWAY:
            retry_run(
                f"kubectl create -f ../resources/seldon-gateway.yaml -n {namespace}"
            )
        retry_run(f"kubectl apply -f ../resources/graph1.json -n {namespace}")
        wait_for_status("mymodel", namespace)
        wait_for_rollout("mymodel", namespace)
        logging.warning("Initial request")
        r = initial_rest_request("mymodel", namespace, endpoint=api_gateway)
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        retry_run(f"kubectl apply -f ../resources/graph3.json -n {namespace}")
        r = initial_rest_request("mymodel", namespace, endpoint=api_gateway)
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        i = 0
        for i in range(100):
            r = rest_request_ambassador("mymodel", namespace, api_gateway)
>           assert r.status_code == 200
E           assert 503 == 200
E             -503
E             +200

test_rolling_updates.py:84: AssertionError
------------------------------ Captured log setup ------------------------------
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl create namespace test-rolling-update2-ambas
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl apply -f ../resources/graph1.json -n test-rolling-update2-ambas
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:115 Status for SeldonDeployment mymodel is ready.
WARNING  root:seldon_e2e_utils.py:52 Successfully waited for SeldonDeployment mymodel
WARNING  root:seldon_e2e_utils.py:63 For SeldonDeployment mymodel found following deployments: ['mymodel-mymodel-e2eb561']
WARNING  root:seldon_e2e_utils.py:76 Waiting for deployment mymodel-mymodel-e2eb561
WARNING  root:seldon_e2e_utils.py:83 Successfully waited for deployment mymodel-mymodel-e2eb561
WARNING  root:test_rolling_updates.py:73 Initial request
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl apply -f ../resources/graph3.json -n test-rolling-update2-ambas
_________________ TestRollingHttp.test_rolling_update7[ambas] __________________
[gw0] linux -- Python 3.6.0 /usr/local/bin/python

self = <test_rolling_updates.TestRollingHttp object at 0x7f17abb6ea58>
namespace = 'test-rolling-update7-ambas', api_gateway = 'localhost:8003'

    @with_api_gateways
    # test changing the image version and the name of its container
    def test_rolling_update7(self, namespace, api_gateway):
        if api_gateway == API_ISTIO_GATEWAY:
            retry_run(
                f"kubectl create -f ../resources/seldon-gateway.yaml -n {namespace}"
            )
        retry_run(f"kubectl apply -f ../resources/graph1svc.json -n {namespace}")
        wait_for_status("mymodel", namespace)
        wait_for_rollout("mymodel", namespace, expected_deployments=2)
        logging.warning("Initial request")
        r = initial_rest_request("mymodel", namespace, endpoint=api_gateway)
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        retry_run(f"kubectl apply -f ../resources/graph3svc.json -n {namespace}")
        r = initial_rest_request("mymodel", namespace, endpoint=api_gateway)
        assert r.status_code == 200
        assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
        i = 0
        for i in range(100):
            r = rest_request_ambassador("mymodel", namespace, api_gateway)
>           assert r.status_code == 200
E           assert 504 == 200
E             -504
E             +200

test_rolling_updates.py:252: AssertionError
------------------------------ Captured log setup ------------------------------
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl create namespace test-rolling-update7-ambas
------------------------------ Captured log call -------------------------------
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl apply -f ../resources/graph1svc.json -n test-rolling-update7-ambas
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:118 Failed to find status - sleeping
WARNING  root:seldon_e2e_utils.py:115 Status for SeldonDeployment mymodel is ready.
WARNING  root:seldon_e2e_utils.py:52 Successfully waited for SeldonDeployment mymodel
WARNING  root:seldon_e2e_utils.py:63 For SeldonDeployment mymodel found following deployments: ['mymodel-mymodel-e2eb561', 'mymodel-mymodel-svc-orch-8e2a24b']
WARNING  root:seldon_e2e_utils.py:76 Waiting for deployment mymodel-mymodel-e2eb561
WARNING  root:seldon_e2e_utils.py:83 Successfully waited for deployment mymodel-mymodel-e2eb561
WARNING  root:seldon_e2e_utils.py:76 Waiting for deployment mymodel-mymodel-svc-orch-8e2a24b
WARNING  root:seldon_e2e_utils.py:83 Successfully waited for deployment mymodel-mymodel-svc-orch-8e2a24b
WARNING  root:test_rolling_updates.py:241 Initial request
WARNING  root:seldon_e2e_utils.py:98 Successfully ran command: kubectl apply -f ../resources/graph3svc.json -n test-rolling-update7-ambas
=================== 2 failed, 34 passed in 964.57s (0:16:04) ===================
make: *** [Makefile:82: test] Error 1
kind delete cluster
Deleting cluster "kind" ...
Stopping Docker: dockerProgram process in pidfile '/var/run/docker-ssd.pid', 1 process(es), refused to die.
[31m
Pipeline failed on stage 'end-to-end' : container 'step-test-end-to-end'. The execution of the pipeline has stopped.[0m
